Providing the way how a page type can be created for a new route (#6241)
magento/devdocs: src/guides/v2.3/extension-dev-guide/routing.md

![Routing Result]({{ site.baseurl }}/common/images/routing-result.png)

## Declaring the new route as Page Type

> After creating a new route `routing/index/index`, it's a good practice to give more control on it for the admin. By creating a new `Page Type`, the admin can manage the content of this page using widgets. 

Reviewer: ```suggestion
After creating a new route `routing/index/index`, it is a good practice to give admins more control over it. By creating a new `Page Type`, an admin can manage the content of this page using widgets. 
```

Defining a new page type:

> `etc/frontend/page_types.xml`

```xml
<?xml version="1.0"?>
<page_types xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_types.xsd">
    <type id="routing_index_index" label="Routing Page"/>
</page_types>
```

{:.bs-callout-info}
The `routing_index_index` represents the path for the new route.

### Result

> As result, the new page is available in `Specified Page` dropdown, on widget creating page.

Reviewer: This is a little unclear. Is this rewrite accurate?

> As result, the new page is available in the `Specified Page` dropdown widget.

Reviewer 2: Hi @jeff-matthews. I believe it is. He means the following section on the widget creation page

![image](https://user-images.githubusercontent.com/2148959/71187427-5d50f880-227f-11ea-81ec-06f513f3a6d9.png)

Author: Yes, that's right.
